Dockerfile.assisted-service-build

```diff
@@ -23,7 +23,7 @@ RUN pip3 install boto3==1.13.14 waiting==1.4.1 requests==2.22.0 mkdocs==1.1.2 \
 RUN curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.10.1/minikube-linux-amd64 \
   && chmod +x minikube && mkdir -p /usr/local/bin/ && install minikube /usr/local/bin/
 RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl && chmod +x ./kubectl && mv ./kubectl /usr/local/bin/kubectl
-RUN curl -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.6.0/openshift-client-linux-4.6.0.tar.gz | tar -xz -C /usr/local/bin
+COPY --from=quay.io/openshift/origin-cli /usr/bin/oc /usr/bin
```

Comment:
Add tag

Reply:
Actually, I think we should use the methodology we just changed in [assisted-test-infra](https://github.com/openshift/assisted-test-infra/pull/592).
Instead of downloading these programs to the image, and just make it bigger and require more maintenance - mounting them from the host might be easier. Especially when those programs are common to be installed on our hosts.
We have this script initializing the environment on test-infra - https://github.com/openshift/assisted-test-infra/blob/master/scripts/install_minikube.sh

Reply:
Unless I'm missing something, this image seems to be a "utility" (considering it even installs `minikube`, in which case I'm not as worried about the size or it having the binaries in it. Please, correct me if I'm wrong.

One point in favor of having it inside the image is that it makes it easier to create development environments, and test environments as the requirements are fewer. 

I'm ok either way, tbh. Just wanted to bring this up for the sake of discussion :smile:

Reply:
You are right. This "build" image is used solely for a development environment. And still, I would prefer that it would work smoothly and faster :) 
There's a point regarding fixed program versions. But we don't use any special features from `kubectl`, `minikube` nor `oc` so I don't think it's necessary.

You are right that having it all inside the build container is very friendly. But when there would come a time you need to debug your cluster, you would need those tools anyway :) (as opposed to `pip` and `golint`, for example, that aren't debugging tools).